conf/test.config

```diff
@@ -26,5 +26,9 @@ params {
     input  = 'https://raw.githubusercontent.com/nf-core/raredisease/dev/assets/samplesheet.csv'
 
     // Genome references
-    fasta = 'https://github.com/nf-core/test-datasets/raw/sarek/reference/chr20_hg38/Homo_sapiens_assembly38_chr20.fasta'
+    //fasta = 'https://github.com/nf-core/test-datasets/raw/sarek/reference/chr20_hg38/Homo_sapiens_assembly38_chr20.fasta'
+    fasta = "${params.local_genomes}/GRCh38.primary_assembly.genome.fa"
+
+    // Variant catalog file
+    variant_catalog = 'assets/variant_catalog_grch38.json'
```

Comment:
```suggestion
    fasta = 'https://github.com/nf-core/test-datasets/raw/sarek/reference/chr20_hg38/Homo_sapiens_assembly38_chr20.fasta'    

    // Variant catalog file
    variant_catalog = 'https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/testdata/repeat_expansions.json'
```

Reply:
I designed a variant catalog file for our testdata reference fasta. It might need to be updated when we use a new test reference but for now this will suffice.